server/jetstream_cluster.go

```diff
@@ -487,10 +487,17 @@ func (js *jetStream) setupMetaGroup() error {
 	// but do not form our own.
 	if ln := s.getOpts().LeafNode; len(ln.Remotes) > 0 {
 		sys := s.SystemAccount().GetName()
+	ENDFOR:
 		for _, r := range ln.Remotes {
 			if r.LocalAccount == sys {
+				for _, denySub := range r.DenyImports {
+					if subjectIsSubsetMatch(denySub, "$NRG.>") {
```

Comment:
Make const that loves in raft.go so we don't break things later on?

Reply:
fixed.